auth/store.go

```diff
@@ -706,6 +706,10 @@ func (as *authStore) isOpPermitted(userName string, revision uint64, key, rangeE
 		return nil
 	}
 
+	if userName == "" {
+		return ErrUserEmpty
```

Comment:
@heyitsanthony I did not see your last comment. Yeah. I feel EmptyUser should be returned be getUser.